src/main/java/com/iridium/iridiumskyblock/Item.java

```diff
@@ -4,57 +4,115 @@
 
 import java.util.List;
 
+/**
+ * Class which represents an item in an inventory.
+ * Uses {@link XMaterial} for multi-version support.
+ */
 public class Item {
 
     public XMaterial material;
     public int amount;
-    public String title;
+    public String displayName;
     public String headData;
     public String headOwner;
     public List<String> lore;
     public Integer slot;
 
+    /**
+     * Default, empty constructor.
+     * No one knows why it exists in the first place.
```

Comment:
Because the Jackson serializer needs a no args constructor for it to load stutff